src/main/webapp/app/guided-tour/guided-tour.component.html

```diff
@@ -22,15 +22,20 @@ <h3 jhiTranslate="{{ currentTourStep.headlineTranslateKey }}" class="headline" *
             <div class="tour-block__content">
                 <h5 jhiTranslate="{{ currentTourStep.subHeadlineTranslateKey }}" class="subHeadline" *ngIf="currentTourStep.subHeadlineTranslateKey"></h5>
                 <span jhiTranslate="{{ currentTourStep.contentTranslateKey }}"></span>
-                <div *ngIf="currentTourStep.externalUrl && currentTourStep.externalUrlTranslateKey">
-                    <a jhiTranslate="{{ currentTourStep.externalUrlTranslateKey }}" [href]="currentTourStep.externalUrl" target="_blank"></a>
-                </div>
-                <div *ngIf="currentTourStep.contentType === ContentType.IMAGE && currentTourStep.imageUrl">
+                <div *ngIf="currentTourStep.imageUrl">
                     <img src="{{ currentTourStep.imageUrl }}" />
                 </div>
-                <div *ngIf="currentTourStep.contentType === ContentType.VIDEO && currentTourStep.videoUrl">
+                <div *ngIf="currentTourStep.videoUrl">
```

Comment:
We have the `type-check.pipe.ts` for type checking in the templates.

Reply:
@thilo-behnke I would have to extend the `type-check.pipe.ts`since it currently only takes an Array as input and then have all tour types listed as properties in the `GuidedTourComponent` to do the type check. Isn't the current approach easier to understand?

Reply:
You could also provide a new type check pipe that works with a single object.
I would consider your approach unsafe as the existence of an attribute doesn't safely determine the classType of an object. If you e.g. at a later time re-arrange the attributes you might not notice that you need to adjust this check in the template.

Reply:
@thilo-behnke good hint, let's add a new type check pipe in another PR then. I think this is fine for now.